download/alpha.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: base
+title: Alpha Download
+navgroup: download
+redirect_from:
+  - /alpha/
+  - /Alpha/
+---
+<div class="row">
+  <div class="small-12 columns">
+    <h1>Download</h1>
+
+    <h5>The alpha MonoDevelop release is: <strong>{{ site.data.alpharelease.version }}</strong></h5>
+    <p>Please choose your operating system to view the available packages. Source code is available on <a href="https://github.com/mono/monodevelop">GitHub</a> (viewing only) or as a <a href="https://download.mono-project.com/sources/monodevelop/">Tarball</a>.</p>
+
+    <dl id="mono-download" class="tabs horizontal" data-tab data-options="deep_linking: true; scroll_to_content: false">
+      <dd class="active"><a href="#download-mac"><i class="fa fa-apple"></i>&nbsp;Mac OS X</a></dd>
+      <dd><a href="#download-lin"><i class="fa fa-linux"></i>&nbsp;Linux</a></dd>
+      <dd><a href="#download-win"><i class="fa fa-windows"></i>&nbsp;Windows</a></dd>
+    </dl>
+
+    <div class="tabs-content" id="mono-download-panel">
+      <div class="panel content active" id="download-mac" style="padding: 20px 10px 10px 10px">
+        <h3>Visual Studio for Mac is available as a Mac Package (.pkg)</h3>
+        <div>
+          <a href="{{ site.data.alpharelease.monodevelop_mac_url }}" class="button radius"><i class="fa fa-download"></i> Download Visual Studio for Mac</a>
+          <p>Supported on Mac OS X 10.7 and later.</p>
+        </div>
+      </div>
+      <div class="panel content active" id="download-lin" style="padding: 10px 10px 10px 10px">
+        <h3>MonoDevelop for Linux is available as a Flatpak package (.flatpakref)</h3>
+        <div>
+          <a href="{{ site.data.alpharelease.monodevelop_linux_url }}" class="button radius"><i class="fa fa-download"></i> Download MonoDevelop</a>
+          <p>Supported on any distribution with <a href="https://flatpak.org">Flatpak</a> 0.8.3+</p>
+          <hr />
+          <p>Currently this download only works on x86-64 operating systems. We plan to offer x86 and ARM64 builds in the future.</p>
+        </div>
+      </div>
+      <div class="panel content" id="download-win" style="padding: 20px 10px 10px 10px">
+        <h3>MonoDevelop for Windows is available from source only</h3>
+        <p>Please refer to the <a href="/developers/building-monodevelop">building guide</a> for more information about how to install and configure your MonoDevelop.</p>
+        <hr />
+        <h5>GTK# for .NET</h5>
```

Comment:
remove this or point to http://www.mono-project.com/download/#download-win so we don't need to update it in two places (applies to beta/stable too)